setup.py

```diff
@@ -25,8 +27,7 @@ def get_version():
         'pytz>=2016.4',
         'graphviz~=0.4',
         'jsonschema~=2.5',
-        'psqlgraph~=2.0',
-        'dictionaryutils>=1.2.0,<3.0.0',
+        'psqlgraph',
```

Comment:
why did you remove the pin?

Reply:
I think because running python setup.py install would overwrite the pipenv installed version of psqlgraph with the one from pypi. We should probably pin it once we release py3 psqlgraph to pypi.